extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -561,11 +564,11 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
                 // check that this instance is not not used by ajax call performed inside client side monitoring to send data to collector
                 if (!isNullOrUndefined(xhr)) {
                     // Look on the XMLHttpRequest of the URL string value
-                    isDisabled = xhr[DisabledPropertyName] === true || theUrl[DisabledPropertyName] === true;
+                    isDisabled = xhr[DisabledPropertyName] === true || theUrl[DisabledPropertyName] === true || xhr[_excludeRequestFromAutoTrackingPropertyName] === true || theUrl[_excludeRequestFromAutoTrackingPropertyName] === true;
```

Comment:
@MSNev hi Nev, can you please explain what does `theUrl[DisabledPropertyName]` mean? Both `theUrl` and `DisabledPropertyName` are string, one string will never has a string property isn't it? Should we remove both the checks here? And my IDE complains this line also: 
![image](https://user-images.githubusercontent.com/20292261/122136484-73e5fa80-cdf7-11eb-968b-671c4afb27bf.png)